src/20_Components/amp-video-iframe.html

```diff
@@ -0,0 +1,87 @@
+  "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  With [`amp-video-iframe`](https://www.ampproject.org/docs/reference/components/amp-video)
+  you can include a custom-built video player that will obtain all the features available
+  in the [AMP Video Interface](https://github.com/ampproject/amphtml/blob/master/spec/amp-video-interface.md)
+
+  This example integrates an AMP page with a simple custom player that uses the [Video.JS](https://videojs.com/)
+  framework.
+
+  You can use any framework or custom-built player by following the integration instructions in
+  [the component documentation](https://www.ampproject.org/docs/reference/components/amp-video-iframe)
+
+  You can see the sample in action [here](preview).
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-video-iframe</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <!-- ## Include the amp-video-iframe extension script. -->
+  <script async custom-element="amp-video-iframe" src="https://cdn.ampproject.org/v0/amp-video-iframe-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+  <!--
+    ## Include the `amp-video-iframe` tag.
+    Set the `autoplay` attribute to start automatically as the user scrolls the
+    video into view.
+
+    The `src` attribute will point to a document including a video player. This
+    URL must be relative or have HTTPS defined.
+
+    You can read the source for the document referenced
+[here](https://github.com/ampproject/amp-by-example/blob/master/static/amp-video-iframe-videojs.html).
+  -->
+
+  <amp-video-iframe
+      id="myVideo"
+      src="/amp-video-iframe-videojs.html"
+      width="720"
+      height="405"
+      layout="responsive"
+      autoplay>
+    <div placeholder style="background: black; color: white">
+      This is a placeholder
+    </div>
+  </amp-video-iframe>
+
+  <!--
+    ## Integration
+    Inside the document in your frame, you must include a script and a simple
+    integration callback in order for the host to comunicate with your frame.
+
+    <div class="abe-code-container">
```

Comment:
@sebastianbenz Not sure what the best way to include code here would be.

This code is not supposed to be rendered in the same document. Instead, it refers to code living inside the iframe. It would be nice if we could include it and get syntax highlighting somehow.

![image](https://user-images.githubusercontent.com/254946/49477324-6afd2280-f7d1-11e8-91d5-edfc756ee139.png)

Reply:
Awesome, thanks!